lms/djangoapps/certificates/models.py

```diff
@@ -263,9 +263,17 @@ class GeneratedCertificate(models.Model):
     # results. Django requires us to explicitly declare this.
     objects = models.Manager()
 
-    MODES = Choices(u'verified', u'honor', u'audit', u'professional', u'no-id-professional', u'masters')
+    MODES = Choices(
+        'verified',
+        'honor',
+        'audit',
+        'professional',
+        'no-id-professional',
+        'masters',
+        'professional-education'
```

Comment:
`professional-education` here, but `executive-education` everywhere else? Was this a typo or intended to be different?

Reply:
:facepalm: You saved the day. Some part of my brain keeps calling it Prof Ed and some part of my project manager keeps correcting me. Will fix :) :+1: